trollmoves/client.py

```diff
@@ -407,6 +407,10 @@ def reload_config(filename, chains, callback=request_push, pub_instance=None):
             if val.get("heartbeat", False):
                 topics.append(SERVER_HEARTBEAT_TOPIC)
             for provider in chains[key]["providers"]:
+                if '/' in provider.split(':')[-1]:
+                    parts = urlparse(provider)
+                    provider = parts.scheme + '://' + parts.netloc
```

Comment:
use `urlunparse` for this.

Reply:
Using `urlunparse` would be unnecessarily complicated as the attributes can't be modified, and the topic (`parts.path`) needs to be removed from the provider.

Reply:
I insist :) I think it'll make the code more robust.